src/Makefile.am

```diff
@@ -30,4 +30,4 @@ nobase_include_HEADERS = \
         ../include/opendht/dhtrunner.h \
         ../include/opendht/default_types.h \
         ../include/opendht/rng.h \
-        ../include/opendht/indexation/pht.h
+		../include/opendht/indexation/pht.h
```

Comment:
Tab issues. Since I know you use `vim`, you should make sure to `:set noexpandtab` before inserting a tab in an automake file.